fiftyone/server/metadata.py

```diff
@@ -87,10 +85,6 @@ async def get_metadata(
                     aspect_ratio=width / height,
                     frame_rate=frame_rate,
                 )
-        elif has_orthographic_projection_metadata:
```

Comment:
I think this is still needed: when the projection images are being rendered in the grid, you need to use them to compute the aspect ratio so that the grid tiling algorithm works properly.

As currently implemented, the tiling is not correct in the grid.

```py
import fiftyone as fo
import fiftyone.utils.utils3d as fou3d
import fiftyone.zoo as foz

dataset = foz.load_zoo_dataset(
    "quickstart-groups",
    dataset_name="test-quickstart-groups",
    max_samples=30,
)
dataset.group_slice = "pcd"

fou3d.compute_orthographic_projection_images(dataset, (-1, 512), "/tmp/projections")

session = fo.launch_app(dataset)
```

Reply:
Yeah, I got up this morning with the same thought in my head 😅  pushed a fix in the new commit